docs/content/support/v18-migration.mdx

```diff
@@ -27,7 +27,7 @@ See [color utility classes](/utilities/colors) for a list of all the functional
 | `.color-text-inverse`   | `.color-fg-on-emphasis` |
 | `.color-text-white`     | n/a*                    |
 
-* Often `.color-text-white` can be replaced with `.color-fg-on-emphasis`, but there are some cases where that's not ideal. If in doubt, plase reach out the the `#primer` team.
+* Often `.color-text-white` can be replaced with `.color-fg-on-emphasis`, but there are some cases where that's not ideal. If in doubt, please reach out to the `#primer` team.
```

Comment:
The second change here was caught by duplicate word detection. if it was a two letter word, e.g. `th` or `te`, the spell checker would, as currently tuned, ignore it, as the current threshold is 3 letters.